include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -2579,7 +2581,7 @@ struct OrtApi {
    * This will create the configuration of an arena that can eventually be used to define an arena based allocator's behavior
    *
    * \param[in] max_mem Use 0 to allow ORT to choose the default
-   * \param[in] arena_extend_strategy Use -1 to allow ORT to choose the default, 0 = kNextPowerOfTwo, 1 = kSameAsRequested
+   * \param[in] arena_extend_strategy Use -1 to allow ORT to choose the default, 0 = kNextPowerOfTwo, 1 = kSameAsRequested, 2 = kLimitedNextPowerOfTwo
```

Comment:
do we really need a new strategy? it looks more like adding a configuration to power of 2 strategy.

Reply:
It's fine to add a config only.
Note that python API does not expose the configuration max_limited_power_of_two_bytes.

Please update the documentation about the design change. Note that max_limited_power_of_two_bytes is not actual limit. When requested bytes is 2GB, allocator will expand 2GB. It is just a limit for expand when requested bytes < 1GB.

Reply:
+1. No new strategy needed. Just add a config.

Reply:
Thanks for all your comments. I have removed the new strategy but only used the new configuration. The python API has expose the new configuration. 
@tianleiwu I'm not sure whether I have updated all the documentations. Please take a look~